### PR TITLE
added admin client list route through server to redux to dom

### PR DESCRIPTION
**Pull request by demkantor**
client list for admin route is running from dom through redux to server to db and back"

### CODE REVIEW

File: server/modules/pool.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ if (process.env.DATABASE_URL) {
   };
 } else {
   config = {
-    // user: null || 'con',
-    // password: null || 'secretpass',
+    user: null || 'con',
```
Comment (haley-r) [inline server/modules/pool.js:24]: i might just be blanking, but what is this for?

**Comment by haley-r** [comment]
besides the one note I left, things make sense/ look good. did not pull down to test or look suuuuper closely. more commits would be nice!

File: server/modules/pool.js (hunk 1 above)
Comment (demkantor) [reply server/modules/pool.js:24]: oh thats just for my use in databses, i have to give stuff username and passwords, do my best to comment it out before i send anything to github so no one has to figure out what it is

Comment (haley-r) [reply server/modules/pool.js:24]: oh cool, that rings a bell-thanks for clarifying, if it ever makes it through a pull I'll know what to do

**Pull request merged**